examples/fixed_column_width.html

```diff
@@ -0,0 +1,163 @@
+<!--
+   
+   Copyright (c) 2020, the Regular Table Authors.
+   
+   This file is part of the Regular Table library, distributed under the terms of
+   the Apache License 2.0.  The full license can be found in the LICENSE file.
+
+-->
+
+<!--
+    This is an example of how to set up columns with fixed width just using
+    Javascript and CSS rules.
+
+    The default behaviour that regular-table implements for this feature includes:
+    * The default width of the cells corresponding to a column is calculated
+    taking into account the size of the content of the visible cells for that column.
+    * It is only possible to increase the width of a cell up to the size of the
+    content currently visible for that column.
+    * It is possible to manually shrink the column width up to the limit of 10 pixels.
+    * Column widths are calculated by the library using the max-width css rule. Which
+    means that settings the max-width from a css rule will lead to a fixed width
+    behavior for the cells of that column.
```

Comment:
I believe these list-item hard wraps need to be indented https://www.markdownguide.org/basic-syntax/#adding-elements-in-lists